modules/stanford_bean_types_permissions/stanford_bean_types_permissions.info

```
@@ -7,26 +7,34 @@ dependencies[] = bean_admin_ui
 dependencies[] = ctools
 dependencies[] = features
 dependencies[] = stanford_bean_types
+dependencies[] = stanford_jumpstart_roles
```

We don't want to make this a dependency, as stanford_bean_types is on the stack.

I think what will happen if you do not have a role defined (which you may not have "editor" or "site owner" if you don't have stanford_jumpstart_roles), it will just fail silently, which is fine in this case.

Permissions Features are a pain in the butt.